numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -4232,3 +4232,35 @@ def add_newdoc(place, name, doc):
     array([ 0, 20, 20, 60, 20, 20])
 
     """)
+
+add_newdoc('numpy.core.umath', 'countbits',
+    """
+    Computes the number of 1-bits in ``x``.
+
+    Parameters
+    ----------
+    x : array_like, unsigned int
+        Input array.
+    $PARAMS
+
+    Returns
+    -------
+    y : ndarray
+        The corresponding number of 1-bits in the input.
+        $OUT_SCALAR_1
+
+    References
+    ----------
+    .. [1] https://stackoverflow.com/a/109025/5671364
+
+    .. [2] Wikipedia, "Hamming weight",
+           https://en.wikipedia.org/wiki/Hamming_weight
+
+    Examples
+    --------
+    >>> np.countbits(np.uint64(1023))
+    10
+    >>> np.countbits(np.arange(10, dtype=np.uint32))
+    array([0, 1, 1, 2, 1, 2, 2, 3, 1, 2], dtype=uint32)
+
+    """)
```

Comment:
I think it's worth including `popcount` somewhere in the docstring, since that seems like something people are likely to be searching for.

Reply:
Oh yeah, makes sense, will get indexed that way